webplugin/js/app/km-message-markup-1.0.js

```diff
@@ -20,4 +20,17 @@ Kommunicate.messageTemplate = {
         var data = {key: key};
         return Mustache.to_html(Kommunicate.messageTemplate.getProgressMeter(), data);
     }
+};
+
+Kommunicate.popupChatTemplate = {
+
+    getPopupChatTemplate: function(popupWidgetContent) {
+        var index = (popupWidgetContent.length && popupWidgetContent[0].templateKey) || KommunicateConstants.CHAT_POPUP_TEMPLATE.HORIZONTAL;
+        var chatPopupTemplateMarkup = {
+            1: '<div id="chat-popup-widget-container" class="chat-popup-widget-container chat-popup-widget-container--horizontal n-vis applozic-launcher"><div class="chat-popup-widget-text-wrapper"><p class="chat-popup-widget-text">' + (popupWidgetContent.length && popupWidgetContent[0].message) + '</p></div></div>',
```

Comment:
@CaptainT33mo is it same other than 'horizontal' vs 'vertical'?

Reply:
@devashishmamgain Yes, they are the same. Currently removed the separate markup and combined them into one.